src/Shared/Program.cs

```diff
@@ -39,7 +41,11 @@ public static void Main(string[] args)
             {
                 listing = ParseListingName(listing);
 
-                Type target = assembly.GetTypes().First(type => type.FullName.Contains(listing));
+                Type target = assembly.GetTypes().First(type =>
+                {
+                    Regex reg = new Regex($"{listing}\\.+");
```

Comment:
Same comment as the other PR. This regex should be declared outside of the First call